src/test/java/com/softserveinc/ita/SampleTest.java

```diff
@@ -1,12 +1,23 @@
-package com.softserveinc.ita;
+package com.softserveinc.ita.vsaroz;
 
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import java.util.concurrent.TimeUnit;
 
 public class SampleTest {
-
+    @BeforeSuite
+    public void beforeSuite() {
+        //Installing chrome driver
+        System.setProperty("webdriver.chrome.driver", "E://SoftServe//chromedriver_win32//chromedriver.exe");
```

Comment:
place a chromedriver in this package 'main/java/com/softserveinc/ita' create here a package 'drivers'